pkg/kv/scan.go

```diff
@@ -43,6 +43,7 @@ func (b *PrefixIterator) Close() {
 }
 
 // ScanPrefix returns an iterator on store that scan the set of keys that start with prefix
+// after is the full key name for which to start the scan from
 func ScanPrefix(ctx context.Context, store Store, prefix, after []byte) (EntriesIterator, error) {
```

Comment:
The behaviour of `after<prefix` is an empty iterator, is this expected?

Reply:
I believe this is the behavior we want to have. The other option is to validate and return an error, but I prefer this option less

Reply:
I think this is a confusing situation. IMO `after<prefix` should behave like `after=""`